ckanext/dcat/profiles.py

```diff
@@ -727,22 +727,25 @@ def _add_list_triples_from_dict(self, _dict, subject, items):
 
     def _add_triples_from_dict(self, _dict, subject, items,
                                list_value=False,
-                               date_value=False):
+                               date_value=False,
+                               multilingual=False):
```

Comment:
`multilingual` is the name of a core extension that does translations with a lookup table in the database. That's not how this code works, so better to use `all_translated` or similar here because you're following the `get_translated` template helper pattern but returning all values IIUC

Reply:
@wardi yeah, just passing the translated dicts into the above mehtod. E.g. `{'en': 'English Value', 'fr': 'French Value'}` instead of just `Value`.

The multilingual is just what Amer suggested, so I can change that easily.